tools/publisher/src/subcommand/publish.rs

```diff
@@ -195,22 +195,36 @@ async fn correct_owner(package: &Package) -> Result<()> {
         Duration::from_secs(5),
         || async {
             let owners = cargo::GetOwners::new(&package.handle.name).spawn().await?;
-            let incorrect_owners = owners.iter().filter(|&owner| owner != CRATE_OWNER);
-            for incorrect_owner in incorrect_owners {
-                cargo::RemoveOwner::new(&package.handle.name, incorrect_owner)
-                    .spawn()
-                    .await
-                    .context("remove incorrect owner")?;
-                info!(
-                    "Removed incorrect owner `{}` from crate `{}`",
-                    incorrect_owner, package.handle
-                );
+            let mut added_individual = false;
+            for &crate_owner in CRATE_OWNERS {
+                if !owners.iter().any(|owner| owner == crate_owner) {
```

Comment:
can just use `owners.contains` here I think

Reply:
Would need a reference to a `String` type:
```
error[E0308]: mismatched types
   --> src/subcommand/publish.rs:200:37
    |
200 |                 if !owners.contains(crate_owner) {
    |                                     ^^^^^^^^^^^ expected struct `std::string::String`, found `str`
    |
    = note: expected reference `&std::string::String`
               found reference `&str`
```